internal/api.py

```diff
@@ -181,6 +208,10 @@ def raidbots(api_key, profile_location, simc_build, output_location, report_name
             sim_data = retrieve_data(api_url_base, sim_id, 'data.full.json')
         output_file = open(output_location, 'w')
         output_file.write(json.dumps(sim_data))
-        print("Saved results to {0}".format(output_location))
+        print(f"Saved results to {output_location}")
     else:
         print("Error getting data from Raidbots")
+
+
+if __name__ == "__main__":
+    poll_status("https://raidbots.com", "7dRu7se1Hbh3K3EAUneWDH")
```

Comment:
I'm assuming this is what you are doing to test locally, make sure we remove this before we merge